examples/redis-demo/grails-app/conf/logback.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Per https://github.com/grails/grails-doc/issues/907 should the file name be changed?

Reply:
Yes, proceed with changing it to logback-spring.xml